controls/color-palette/README.md

```diff
@@ -0,0 +1,45 @@
+# Using the Color Palette
```

Comment:
Make control title names #Color Palette Control for WordPress Customizer and similar